src/status_im/contexts/wallet/common/utils/networks_test.cljs

```diff
@@ -20,3 +20,21 @@
            constants/arbitrum-mainnet-chain-id))
     (is (= (utils/network->chain-id {:network :arbitrum :testnet-enabled? true :goerli-enabled? false})
            constants/arbitrum-sepolia-chain-id))))
+
+(deftest test-short-names->network-preference-prefix
```

Comment:
@mmilad75, thanks for covering the code with unit tests! I think we can further improve this code.

1. In our codebase, but also in many other Clojure repos out there, the name of the test var is suffixed with the `test` word, not prefixed. It's just a convention.
2. The `testing` macro is unnecessary because we already know which function is under test based on the test var, which in our repo tend to be the name of the function under test followed by `-test`. Therefore, we try to use the `testing` macro for the cases where we actually need to give more context about the test.
3. (Totally optional, but you may prefer) You can make the tests more concise using the `are` macro. This macro is incredibly useful for testing these pure functions with a small input & output, which is the case of many utility functions. It helps make testing and adding examples cheap and if it's cheap, the greater likelihood devs will test. And even nicer, zprint will autoformat in tabular form. For example (untested):

```clojure
(deftest short-names->network-preference-prefix-test
  (are [expected short-names]
   (= expected (utils/short-names->network-preference-prefix short-names))
   "eth:"          ["eth"]
   "eth:opt:"      ["eth" "opt"]
   "eth:opt:arb1:" ["eth" "opt" "arb1"]))
```

I see you're following the pattern in the file that existed before your PR, but if you could apply the suggestions above, that would be a nice bonus :)

Reply:
Additionally, the `are` macro for this sort of utility function makes adding examples so cheap that it becomes more obvious to me that the examples are lacking more *sad path* examples or *nil/empty cases* because this is where many implementations fail. Example inputs: `""`, `":"`, `"  "`, and of course `nil`.

Reply:
Done